dataland-frontend/src/components/forms/parts/kpiSelection/EuTaxonomyKPIsModel.ts

```diff
@@ -188,18 +188,18 @@ export const euTaxonomyKpiInfoMappings = {
   alignedRevenue:
```

Comment:
ConvertToMillions wird auch noch irgendwie von lksg benutzt. Es ist mir dabei aus dem Kontext nicht klar, ob das so sein soll.

Reply:
Die description wirkt eigentlich nicht so

Reply:
Ja.  Es gibt im lksg framework ebenfalls ein Feld "total revenue", und dessen Wert wird auf der view page in MM angegeben.   Also 5000000 im dataset wird als 5 MM angezeigt.